jaxrs/WFLY-12192_optional_param.adoc

```diff
@@ -0,0 +1,66 @@
+= WFLY-12192 Add support for injecting Optional<T> parameter types
+:author:            Wei Nan Li
+:email:             weli@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
```

Comment:
@liweinan Can you please updates these default keywords?

Reply:
ok!

Reply:
@marekkopecky Sorry for the late reply. I was moving to some other tasks and let me come back to focus on this one now. For this one I have question: I checked other documents: https://github.com/wildfly/wildfly-proposals/tree/master/jaxrs
And they seems to contain the same keywords as above. I also checked the pr like this:
https://github.com/wildfly/wildfly-proposals/pull/233/files
And it contains similar keywords. Could you please let me know what should I change about these keywords? Thanks!

Reply:
Well, so which component is this this RFE related with? Possible keywords could be `RESTEasy`, `JAX-RS`, `Injection`, etc.

Reply:
good advice! Let me update the tags.